<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="settings">
      <property name="title">
       <string>Question Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="style">
         <property name="title">
          <string>Quiz Style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="qs_custom">
            <property name="text">
             <string>Custom</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_style</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qs_random">
            <property name="text">
             <string>Random</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_style</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qs_millionaire">
            <property name="text">
             <string>Millionaire</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_style</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="customize">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="1">
           <widget class="QSpinBox" name="amount">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_amount">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_categories">
            <property name="text">
             <string>Category</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="categories"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="qtype">
         <property name="title">
          <string>Type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="qt_any">
            <property name="text">
             <string>Any</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_type</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qt_multiple">
            <property name="text">
             <string>Multiple</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_type</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qt_truefalse">
            <property name="text">
             <string>True/False</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_type</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="qdiff">
         <property name="title">
          <string>Difficulty</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="qd_any">
            <property name="text">
             <string>Any</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_diff</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qd_easy">
            <property name="text">
             <string>Easy</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_diff</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qd_medium">
            <property name="text">
             <string>Medium</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_diff</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="qd_hard">
            <property name="text">
             <string>Hard</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bg_diff</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="getQuestionsLinkButton">
         <property name="text">
          <string>Get Questions</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Question Table</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="questionTable">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>400</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Category</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Difficulty</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Answered?</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="selectedQuestion">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="answerBox">
            <property name="title">
             <string>Answers</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Quiz"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Quiz">
   <property name="text">
    <string>New Quiz</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_diff"/>
  <buttongroup name="bg_style"/>
  <buttongroup name="bg_type"/>
 </buttongroups>
</ui>
